airflow/providers/alibaba/cloud/hooks/oss.py

```diff
@@ -163,7 +161,7 @@ def load_string(self, key: str, content: str, bucket_name: Optional[str] = None)
         try:
             self.get_bucket(bucket_name).put_object(key, content)
         except Exception as e:
-            raise AirflowException(f"Errors: {e}")
+            raise AirflowException(f"Errors: {e}") from e
```

Comment:
Maybe we should just let the underlying error bubble up instead. I can’t think of a merit catching this to coerce into AirflowException.

Reply:
@uranusjr but if we have two errors then may overlap?

Reply:
Not sure what you mean. Could you give an example?

Reply:
@uranusjr i will revert this change, i see what you mean. Commit made now so resolving

Reply:
Actually it’s not exactly what I meant, but this is fine as well.